python/BioSimSpace/Sandpit/Exscientia/Process/_gromacs.py

```diff
@@ -2399,22 +2403,27 @@ def _parse_energy_terms(text):
         U-B, Proper-Dih.. Note that this order is absolute and will not be
         changed by the input to `gmx energy`.
         """
-        sections = text.split("---")
+        # Get rid of all the nasty message from argo
+        content = text.split("End your selection with an empty line or a zero.")[1]
+        sections = content.split("---")
         # Remove the empty sections
-        sections = [section for section in sections if section]
+        sections = [section for section in sections if section.strip()]
         # Concatenate the lines
-        section = sections[1].replace("\n", "")
+        section = sections[0].replace("\n", "")
         terms = section.split()
         # Remove the possible '-' from the separation line
         terms = [term for term in terms if term != "-"]
         # Check if the index order is correct
-        indexes = [int(term) for term in terms[::2]]
+        try:
```

Comment:
is the try necessary here? if it fails, then it's in the _parse_energy_terms function, so it would be clear that the parsing failed,

Reply:
It is more like it is easier to find out what happens in argo.

Reply:
Fair enough, that might be provided in the "local variables" log as well.